addons/project/static/src/project_sharing/project_sharing.js

```diff
@@ -33,9 +33,9 @@ export class ProjectSharingWebClient extends Component {
     }
 
     async _showView() {
-        const { action_name, project_id, open_task_action } = session;
+        const { action, project_id, open_task_action } = session;
```

Comment:
Might be more correct, but logic wise, it is an unnecessary change.
We avoid making those, because they may mess forward-ports up. :(

Reply:
yes makes sense sorry :'(